.github/CODEOWNERS

```diff
@@ -1003,6 +1003,12 @@
 # ServiceLabel: %Large Instance %Mgmt
 # ServiceOwners:                                                   @8Gitbrix
 
+# PRLabel: %Monitor
+/sdk/monitor/Azure.ResourceManager.Monitor/                        @thomasp98296 @ritwik8119
```

Comment:
@thomasp98296:  `ritwik8119` still does not have a `Microsoft` organization membership.